antora-playbook.yml

```diff
@@ -24,7 +24,7 @@ ui:
   bundle:
     url: https://code.asam.net/common/antora-ui/-/jobs/artifacts/main/raw/build/ui-bundle.zip?job=bundle-stable
     snapshot: true
-  supplemental_files: ./supplemental_ui # Activate this if you need to make local UI changes. Check the official Antora documentation for more information.
+# supplemental_files: ./supplemental_ui # Activate this if you need to make local UI changes. Check the official Antora documentation for more information.
```

Comment:
Can they be removed instead of commenting them out?

Reply:
I would leave it as we need the supplemental files to add the deliveries to the header (https://github.com/asam-ev/OpenMATERIAL-3D/issues/346).